src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py

```diff
@@ -1060,15 +1050,14 @@ def readCondorLog(self, job):
             cres = condor.read_events(logfileobj, 1)
             ulog = list(cres)
             if len(ulog) > 0:
-                if all(key in ulog[-1] for key in ("TriggerEventTypeNumber", "QDate", "JobStartDate",
-                                                   "EnteredCurrentStatus", "MATCH_EXP_JOBGLIDEIN_CMSSite",
-                                                   "WMAgent_JobID")):
+                if all(key in ulog[-1] for key in ("TriggerEventTypeNumber", "QDate", "EnteredCurrentStatus",
```

Comment:
I think the best we can do is to get this Ad and if it's not available, fallback to the first site in the `DESIRED_Sites`. Actually, it may spoil  the BossAir db in case it already had the right site information.